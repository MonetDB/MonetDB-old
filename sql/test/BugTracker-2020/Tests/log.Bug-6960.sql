SELECT LOG(2.0, 64.0); -- 6.0
SELECT LOG(2,65536); -- 16.0
SELECT LOG(10,100); -- 2.0
SELECT LOG(2,42); -- 5.392317423
SELECT LOG(1,100); -- error, divide by 0
