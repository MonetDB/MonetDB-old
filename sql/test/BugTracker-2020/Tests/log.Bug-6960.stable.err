stderr of test 'log.Bug-6960` in directory 'sql/test/BugTracker-2020` itself:


# 18:18:15 >  
# 18:18:15 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-201441" "--port=37404"
# 18:18:15 >  

MAPI  = (monetdb) /var/tmp/mtest-201441/.s.monetdb.37404
QUERY = SELECT LOG(1,100); -- error, divide by 0
ERROR = !Math exception: Divide by zero

# 18:18:15 >  
# 18:18:15 >  "Done."
# 18:18:15 >  

