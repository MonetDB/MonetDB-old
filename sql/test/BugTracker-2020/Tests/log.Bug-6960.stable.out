stdout of test 'log.Bug-6960` in directory 'sql/test/BugTracker-2020` itself:


# 18:18:15 >  
# 18:18:15 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-201441" "--port=37404"
# 18:18:15 >  

#SELECT LOG(2.0, 64.0); -- 6.0
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 6	]
#SELECT LOG(2,65536); -- 16.0
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 16	]
#SELECT LOG(10,100); -- 2.0
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 2	]
#SELECT LOG(2,42);
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 5.392317423	]

# 18:18:15 >  
# 18:18:15 >  "Done."
# 18:18:15 >  

